instrumentation/aws-lambda-1.0/auto/aws-lambda-1.0-auto.gradle

```diff
@@ -21,6 +21,7 @@ muzzle {
     group = "com.amazonaws"
     module = "aws-lambda-java-core"
     versions = "[1.0.0,)"
+    extraDependency('com.amazonaws:aws-lambda-java-events:2.2.1')
```

Comment:
I wonder if I should instead of a separate instrumentation module, `aws-lambda-sqs-2.2.1` due to this extra, optional dependency. Not quite sure where we want to set boundaries for highly related instrumentation like this case, let me know any thoughts.

Reply:
I think it makes sense here to add muzzle requirement on `aws-lambda-java-events` also, e.g.

```
  pass {
    group = "com.amazonaws"
    module = "aws-lambda-java-events"
    versions = "[2.2.1,)"
    extraDependency('com.amazonaws:aws-lambda-java-core:1.0.0')
  }
```